src/business_logic/state/cached_state.rs

```diff
@@ -255,26 +255,28 @@ mod tests {
     }
 
     #[test]
-    fn cached_state_deploy_contract_test() {
+    fn cached_state_deploy_contract_test() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I think there is no need to return a Result in a test. We can just unwrap if we need